cmd/pwmTools/shuffle.go

```diff
@@ -0,0 +1,86 @@
+package main
```

Comment:
I'm not sure what the purpose of this command is. Why do you need it? Functions missing documentation and command missing group and documentation